src/Atoms/Input/Input.jsx

```diff
@@ -30,7 +30,9 @@ const sizes = {
 export const inputStyles = ({ size }) => css`
   &.MuiInputBase-root  {
     .MuiInputBase-input {
-      ${systemCss({ px: 'l', py: 'm', lineHeight: '1', fontSize: 'body' })}
+      ${systemCss({
+    px: 'l', py: 'm', lineHeight: '1', fontSize: 'body', backgroundColor: 'white',

```

Comment:
This makes the input ugly, try to put it on the parent DOM node
`MuiInputBase-root` `MuiInput-root`